src/vm/codeman.cpp

```diff
@@ -5114,7 +5114,7 @@ BOOL NativeImageJitManager::JitCodeToMethodInfo(RangeSection * pRangeSection,
         }
 
         //Get the function entry that corresponds to the real method desc.
-        _ASSERTE(RelativePc >= RUNTIME_FUNCTION__BeginAddress(FunctionEntry));
+        _ASSERTE((RelativePc >= RUNTIME_FUNCTION__BeginAddress(FunctionEntry)));
```

Comment:
Sure thing, I found it strange as well but didn't really think much of it, I think it's because the stringing operator can't handle the spaces.

```
cd /home/ben/git/coreclr/bin/obj/Linux.arm.Debug/src/vm/dac && /usr/bin/clang++-3.5   -DARM -DBETA=0 -DBUILDENV_CHECKED=1 -DCONDITION_HANDLING=1 -DDACCESS_COMPILE -DDBG_TARGET_32BIT=1 -DDBG_TARGET_ARM -DDBG_TARGET_ARM=1 -DDBG_TARGET_ARM_UNIX -DDBG_TARGET_WIN32=1 -DDEBUG -DDEBUGGING_SUPPORTED -DDEV10 -DDEVL=1 -DDISABLE_CONTRACTS -DFEATURE_APPDOMAIN_RESOURCE_MONITORING -DFEATURE_ARRAYSTUB_AS_IL -DFEATURE_ASYNC_IO -DFEATURE_BCL_FORMATTING -DFEATURE_COLLECTIBLE_TYPES -DFEATURE_CORECLR -DFEATURE_COREFX_GLOBALIZATION -DFEATURE_CORESYSTEM -DFEATURE_CORRUPTING_EXCEPTIONS -DFEATURE_DBGIPC_TRANSPORT_DI -DFEATURE_DBGIPC_TRANSPORT_VM -DFEATURE_ENABLE_HARDWARE_EXCEPTIONS -DFEATURE_EXCEPTIONDISPATCHINFO -DFEATURE_FRAMEWORK_INTERNAL -DFEATURE_HOSTED_BINDER -DFEATURE_HOST_ASSEMBLY_RESOLVER -DFEATURE_ICASTABLE -DFEATURE_IMPLICIT_TLS -DFEATURE_ISYM_READER -DFEATURE_LEAVE_RUNTIME_HOLDER=1 -DFEATURE_LEGACYNETCF -DFEATURE_LEGACYNETCFFAS -DFEATURE_LEGACYNETCF_DBG_HOST_CONTROL -DFEATURE_LOADER_OPTIMIZATION -DFEATURE_MAIN_CLR_MODULE_USES_CORE_NAME -DFEATURE_MANAGED_ETW -DFEATURE_MANAGED_ETW_CHANNELS -DFEATURE_MERGE_CULTURE_SUPPORT_AND_ENGINE -DFEATURE_MERGE_JIT_AND_ENGINE -DFEATURE_MULTICOREJIT -DFEATURE_NORM_IDNA_ONLY -DFEATURE_PAL -DFEATURE_PAL_SXS -DFEATURE_PERFMAP -DFEATURE_PREJIT -DFEATURE_READYTORUN -DFEATURE_STANDALONE_SN -DFEATURE_STRONGNAME_DELAY_SIGNING_ALLOWED -DFEATURE_STRONGNAME_MIGRATION -DFEATURE_STUBS_AS_IL -DFEATURE_SVR_GC -DFEATURE_SYMDIFF -DFEATURE_SYNTHETIC_CULTURES -DFEATURE_USE_ASM_GC_WRITE_BARRIERS -DFEATURE_VERSIONING -DFEATURE_WIN32_REGISTRY -DFEATURE_WINDOWSPHONE -DFEATURE_WINMD_RESILIENT -DFX_BRANCH_SYNC_COUNTER_VALUE=0 -DLINUX32 -DNTDDI_VERSION=NTDDI_WIN8 -DNTMAKEENV -DNT_INST=0 -DOFFICIAL_BUILD=0 -DPLATFORM_UNIX=1 -DSTUBLINKER_GENERATES_UNWIND_INFO -DUNICODE -DUNIX_ARM_ABI -DURTBLDENV_FRIENDLY=Checked -DWIN32 -DWIN32_LEAN_AND_MEAN=1 -DWINNT=1 -DWINVER=0x0602 -DWRITE_BARRIER_CHECK=1 -D_ARM_ -D_BLD_CLR -D_CRT_STDIO_ARBITRARY_WIDE_SPECIFIERS -D_DBG -D_DEBUG -D_NEW_SDK=1 -D_SECURE_SCL=0 -D_SKIP_IF_SIMULATOR_ -D_TARGET_ARM_=1 -D_UNICODE -D_WIN32 -D_WIN32_IE=0x0900 -D_WIN32_WINNT=0x0602 -D__LINUX__=1 -Wall -Wno-null-conversion -std=c++11 -nostdinc++ -g -O0 -I/home/ben/git/coreclr/bin/obj/Linux.arm.Debug/src/vm/dac -I/home/ben/git/coreclr/src/vm/dac -I/home/ben/git/coreclr/src/vm -I/home/ben/git/coreclr/src/pal/prebuilt/inc -I/home/ben/git/coreclr/src/pal/inc -I/home/ben/git/coreclr/src/pal/inc/rt -I/home/ben/git/coreclr/src/pal/inc/rt/cpp -I/home/ben/git/coreclr/src/pal/src/safecrt -I/home/ben/git/coreclr/src/inc -I/home/ben/git/coreclr/src/strongname/inc -I/home/ben/git/coreclr/src/inc/winrt -I/home/ben/git/coreclr/src/debug/inc -I/home/ben/git/coreclr/src/debug/inc/arm -I/home/ben/git/coreclr/src/debug/inc/dump -I/home/ben/git/coreclr/src/md/inc -I/home/ben/git/coreclr/src/classlibnative/bcltype -I/home/ben/git/coreclr/src/classlibnative/cryptography -I/home/ben/git/coreclr/src/classlibnative/inc -I/home/ben/git/coreclr/src/gc -I/home/ben/git/coreclr/src/vm/arm    -target armv7-linux-gnueabihf -mthumb -mfpu=vfpv3 -ferror-limit=4096 -Werror -Wno-unused-private-field -Wno-unused-variable -Wno-microsoft -Wno-tautological-compare -Wno-constant-logical-operand -Wno-unknown-warning-option -Wno-invalid-offsetof -Wno-incompatible-ms-struct -Wno-unused-value -Wno-invalid-token-paste -fms-extensions -nostdinc -fPIC -o CMakeFiles/cee_dac.dir/__/codeman.cpp.o -c /home/ben/git/coreclr/src/vm/codeman.cpp
/home/ben/git/coreclr/src/vm/codeman.cpp:5117:9: error: too many arguments
      provided to function-like macro invocation
        _ASSERTE(RelativePc >= RUNTIME_FUNCTION__BeginAddress(FunctionEntry));
        ^
/home/ben/git/coreclr/src/inc/debugmacros.h:55:45: note: expanded from macro
      '_ASSERTE'
  #define _ASSERTE(expr) _ASSERTE_MSG(expr, #expr)
                                            ^
<scratch space>:16:1: note: expanded from here
"RelativePc >= RUNTIME_FUNCTION__BeginAddress(FunctionEntry)"
^
/home/ben/git/coreclr/src/inc/debugmacros.h:44:11: note: macro '_ASSERTE_MSG'
      defined here
  #define _ASSERTE_MSG(expr, msg)                                           \
          ^
/home/ben/git/coreclr/src/vm/codeman.cpp:5117:9: error: use of undeclared
      identifier '_ASSERTE_MSG'
        _ASSERTE(RelativePc >= RUNTIME_FUNCTION__BeginAddress(FunctionEntry));
        ^
/home/ben/git/coreclr/src/inc/debugmacros.h:55:26: note: expanded from macro
      '_ASSERTE'
  #define _ASSERTE(expr) _ASSERTE_MSG(expr, #expr)
                         ^
2 errors generated.
make[2]: *** [src/vm/dac/CMakeFiles/cee_dac.dir/__/codeman.cpp.o] Error 1
make[2]: Leaving directory `/home/ben/git/coreclr/bin/obj/Linux.arm.Debug'
make[1]: *** [src/vm/dac/CMakeFiles/cee_dac.dir/all] Error 2
make[1]: Leaving directory `/home/ben/git/coreclr/bin/obj/Linux.arm.Debug'
make: *** [all] Error 2
```